test/DynamoCoreTests/PythonMigrationAssistantTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Dynamo.PythonMigration.MigrationAssistant;
+using NUnit.Framework;
+using System.Collections.Generic;
+
+namespace Dynamo.Tests
+{
+    public class PythonMigrationAssistantTests
+    {
+        [Test]
+        public void CanMigratePyton2CodeToPython3Code()
+        {
+            // Arrange
+            var expectedPython3Code = @"print(""Hello, Dynamo!"")";
+
+            var python2Code = @"print ""Hello, Dynamo!""";
+
+            var inputNames = new List<string>()
+            {
+                "code"
+            };
+            var inputValues = new List<object>()
+            {
+                python2Code
+            };
+
+            // Act
+            var migratedScript = ScriptMigrator.MigrateCode(inputNames, inputValues, "output");
+
+            // Assert
+            Assert.AreNotEqual(python2Code, migratedScript);
+            Assert.AreEqual(expectedPython3Code, migratedScript);
+        }
+
+        [Test]
+        public void MigrationWillNotChangePython3CompatibleCode()
```

Comment:
let's add another test that confirms that a `python2` script that's been migrated to `3` will not change if we try to migrate the already migrated `3` code again.